libs/styles/styles.css

```diff
@@ -706,16 +706,14 @@ header:not(.global-navigation) ~ main {
  */
 header.global-navigation {
   height: var(--global-height-nav);
-  visibility: hidden;
```

Comment:
Is this actually desired? There seems to be a conflict between this and https://github.com/adobecom/milo/pull/2603/files

Reply:
This is not required. We will revert this change.